e2e/path/zonefile

```diff
@@ -51,4 +51,14 @@ _redirect._.uprecord.path.path.example.com.            IN TXT "v=txtv0;to=https:
 
 
 fallbackupstream.path.path.example.com.                IN A 172.20.10.2
-_redirect.fallbackupstream.path.path.example.com.      IN TXT "v=txtv0;type=path;use=_redirect.wrongupstream.path.path.example.com"
\ No newline at end of file
+_redirect.fallbackupstream.path.path.example.com.      IN TXT "v=txtv0;type=path;use=_redirect.wrongupstream.path.path.example.com"
+
+pathchain.path.path.example.com.                       IN A 172.20.10.2
+_redirect.pathchain.path.path.example.com.             IN TXT "v=txtv0;type=path;to=https://fallback-to.path.path.example.com;code=302"
+_redirect._.pathchain.path.path.example.com.           IN TXT "v=txtv0;type=path;re=/fallback/;to=https://fallback-unknown-path.path.path.example.com"
```

Comment:
Why is the `re=/fallback/` necessary? That wouldn't be hit or?

Reply:
`re=/fallback/` is just a random regex so we can test fallback being triggered when the regex from the chained record doesn't support the request's path.

Reply:
So it does go into the regex path and then fallback. That's an assumption the test is making right as it doesn't really know, if it goes into fallback or just a wildcard rec.To redirect or am I missing something?

Reply:
The test is just trying to see if two path records are chained correctly and if the chained record's regex works on the path or not. And in this case, the test sends a wrong path and expects the chained record to trigger fallback with "to" type. I guess we can consider it a "path > path > rec.To fallback" test.

Reply:
+1